packages/ui/src/ui/utils/index.ts

```diff
@@ -361,3 +363,15 @@ export function printUsageLimit(usage: string, limit: string) {
     const usageNoSuffix = usage.substring(0, lastSpaceIndex);
     return `${usageNoSuffix} / ${limit}`;
 }
+
+export const UNIPIKA_ESCAPED_SETTINGS = {
+    asHTML: false,
+    indent: 0,
+    break: false,
+    showDecoded: true,
+    escapeWhitespace: true,
+};
+
+export function formatTextWithEscapedCharacters(text: string): string {
```

Comment:
Is the function used anywhere?

Reply:
I've drafted this PR for some time, it's not ready for review by now
I've prepared this function to display escaped symbols in some components, which shows path but haven't used yet(